setup.py

```diff
@@ -22,17 +22,14 @@
         'console_scripts': ['morph-tool=morph_tool.cli:cli']},
     license="LGPLv3",
     install_requires=[
+        'bluepyopt>=1.9.37',
         'click>=6.7',
-        'functools32>=3.2;python_version<"3.0"',
         'morphio>=2.3.4',
-        'numpy>=1.14',
         'neurom>=1.4.15',
+        'neuron>=7.8',
+        'numpy>=1.14',
+        'pandas>=1.0.3',
```

Comment:
Can we use a lower version of Pandas here? >= 0.25? IS it necessary to use that high version?

Reply:
Hum, it's best to keep it a the latest major version. What is your underlying thought ?

Reply:
My concern is Spack. They don't have a couple of latest major versions.

Reply:
Good point Aleksei.
It seems BlueBrain/Spack is still at 0.25 but Spack/Spack is at 1.0.3
https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/py-pandas/package.py

So I think we can bump pandas in the Spack BlueBrain fork whenever we will need to.

Reply:
I don't guarantee that it would be easy.